### ISSUE TITLE
Pairing broken

### ISSUE THREAD

**Issue by WeeJeWel** [athombv/com.nest#11]
```
Homey.emit( 'authenticate', {}, function ( err, result ) {
        if ( result.url ) { // cannot read property url of undefined
```

**Comment by WeeJeWel** [comment]
Seems only to happen right after installation from the app store. This can be a problem in Homey itself, I will investigate.

**Comment by RobinBol** [comment]
Difficult to debug, as it has to be installed through the app store. Indeed seems to be an external problem.

**Comment by WeeJeWel** [comment]
Error probably solved in 0.8.35.
